### PR TITLE
95 add examples to read the docs

### PR DESCRIPTION
**Pull request by lcortesg**

### CODE REVIEW

File: eegrasp/eegrasp.py
Comment (aweinstein) [inline eegrasp/eegrasp.py:1]: Are the changes in this file related to this PR?

Comment (lcortesg) [reply eegrasp/eegrasp.py:1]: The new .gitignore entries were borrowed from the [PyGSP2](https://github.com/gsp-eeg/PyGSP2/blob/main/.gitignore) repo in order to prevent it from uploading to GitHub unnecessary files created when the docs build is run locally.

Comment (aweinstein) [reply eegrasp/eegrasp.py:1]: I'm talking about the change in this file: eegrasp.py 

I marked the lines in another comment below.

File: eegrasp/eegrasp.py [hunk 1]
```diff
@@ -73,15 +73,12 @@ def euc_dist(self, pos):
             Dimension of the array is number of channels by number of channels
             containing the euclidean distance between each pair of channels.
         """
-
+        from scipy.spatial import distance_matrix
+        
         distance = np.zeros([pos.shape[0], pos.shape[0]],
                             dtype=np.float64)  # Alocate variable
         pos = pos.astype(float)
-        for dim in range(pos.shape[1]):
```
Comment (aweinstein) [inline eegrasp/eegrasp.py:80]: Are these changes related to the PR?

Comment (lcortesg) [reply eegrasp/eegrasp.py:80]: This change was aimed to solve the same problem presented in [PR45](https://github.com/gsp-eeg/EEGraSP/pull/45) (which can be discarded if this change is approved), but with fewer lines of code, and with minimal code intervention, the solution was proposed by @julioRodino and discussed in a previous meeting as well, without this change the examples that use distance calculation will not run, and neither will the docs.

Comment (aweinstein) [reply eegrasp/eegrasp.py:80]: OK

**Review by aweinstein — APPROVED** [review]
